modules/database/src/std/rec/calcoutRecord.c

```diff
@@ -226,6 +226,7 @@ static long process(struct dbCommon *pcommon)
     struct calcoutRecord *prec = (struct calcoutRecord *)pcommon;
     rpvtStruct *prpvt = prec->rpvt;
     int doOutput;
+    unsigned char    pact=prec->pact;
 
     if (!prec->pact) {
```

Comment:
Since you've just read `prec->pact` into `pact` this line could become `if (!pact) {`. Not sure whether most compilers will do that optimization themselves or not.